src/ecdsa/keys.py

```diff
@@ -225,21 +225,21 @@ def to_der(self):
     def get_verifying_key(self):
         return self.verifying_key
 
-    def sign_deterministic(self, data, hashfunc=None, sigencode=sigencode_string):
+    def sign_deterministic(self, data, hashfunc=None, sigencode=sigencode_string, extra_entropy=b('')):
```

Comment:
the parentheses around apostrophes are unnecessary, `b''` is sufficient

Reply:
I was following other portions of the code. (which use `from six import b` b())

If `b''` is fine by you. I will switch to it.

Reply:
ah, sorry, forgot about six, yes, for consistency it would need to be `b('')`, that being said, this six workaround it is necessary for compatibility with Python 2.5 and earlier, while the oldest officially supported by python-ecdsa is Python 2.6 which can handle the `b''` syntax fine